templates/jobs.yaml

```diff
@@ -573,6 +552,7 @@ spec:
           readOnly: false
         # TODO: Find out default resource limits for this specific job (#5)
         #resources:
+      restartPolicy: OnFailure
```

Comment:
```suggestion
```

This was added in #403. You can see it below on line 584 below.

https://github.com/StackStorm/stackstorm-k8s/blob/fabbea9fb89bce6d093ad718def3e28fcfc47c5e/templates/jobs.yaml#L582-L586

Reply:
one more thing required for this to not fail is to remove the 2 leading whitespaces before FAIL on line 557. unless someone does it before me, I'll submit a PR once my company approves me signing of the CLA

Reply:
Cool. I have a fix for that. I'll push a PR in a few minutes.

Reply:
Fix pushed in #411